### PR TITLE
Make the scheduler event loop buffer size configurable

### LINKED ISSUE #397: Make the scheduler event loop buffer size configurable
**Is your feature request related to a problem or challenge? Please describe what you are trying to do.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by yahoNanJing**
# Which issue does this PR close?

Closes #397.

 # Rationale for this change

Since the channel in the event loop is a bounded one, sometimes when there're many events produced in a short time, it will block the caller when sending events to the channel. If the buffer size is not larger enough, it will downgrade the performance, especially in a high throughput system.

# What changes are included in this PR?

# Are there any user-facing changes?

### CODE REVIEW

**Comment by yahoNanJing** [comment]
Hi @thinkharderdev and @andygrove, could you help review this PR?

File: ballista/scheduler/src/standalone.rs [hunk 1]
```diff
@@ -38,6 +38,7 @@ pub async fn new_standalone_scheduler() -> Result<SocketAddr> {
             "localhost:50050".to_owned(),
             Arc::new(client),
             BallistaCodec::default(),
+            100000,
```
Comment (andygrove) [inline ballista/scheduler/src/standalone.rs:41]: Is it intentionaly to use `100,000` here and not `10,000` like other places?

Comment (yahoNanJing) [reply ballista/scheduler/src/standalone.rs:41]: It's not intentional. It's better to set it to be the same as other places. I'll change it.

**Comment by yahoNanJing** [comment]
For production usage, we set the buffer size to be one million so that large queries with tens of thousands tasks will not be blocked.

**Pull request merged**